README.md

```diff
@@ -23,7 +23,7 @@ The Go client library to interact with the [IBM Cloud Continuous Delivery Toolch
 
 <!-- toc -->
 
-- [IBM Cloud Continuous Delivery Go SDK 1.1.3](#ibm-cloud-continuous-delivery-go-sdk-104)
+- [IBM Cloud Continuous Delivery Go SDK 1.1.3](#ibm-cloud-continuous-delivery-go-sdk-113)
```

Comment:
Might be best to remove the `113` from the anchor link, as I dont think the auto-updater updates that string since it doesnt match the usual format `1.1.3`

Reply:
Hmm, I dont recall making this update, I wonder if its the vscode readme plugin that auto updated it or something. But when I test out the link with `113` it works and if I take it off it doesn't

Reply:
Ah as it has to match the text of the title on [line #1](https://github.com/IBM/continuous-delivery-go-sdk/pull/58/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1). I guess stick with it as is then